Build failing due to xzing kotlin compile

### What
@g123k 
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':scanner_zxing:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 35s
Exception: Gradle task assembleDebug failed with exit code 1
```

--- a/packages/scanner/zxing/android/src/main/kotlin/org/openfoodfacts/scanner_zxing/ZXingUtils.kt
+++ b/packages/scanner/zxing/android/src/main/kotlin/org/openfoodfacts/scanner_zxing/ZXingUtils.kt
@@ -52,7 +52,7 @@ object ZXingUtils {
             )
         )
 
-        return runCathing {
+        return runCatching {
             MultiFormatReader().decode(binaryBitmap).text
         }.getOrNull()
     }

